### PR TITLE
Small Tweaks, Bug Fixes and Docs Improvements

### PR DESCRIPTION
**Pull request by jpjsonic**

### CODE REVIEW

File: mkdocs/topics/sources.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 Sources are data sources. They are databases that contain your data. This page describes some best practices how to set sources up. 
 
 !!! tip
-    There is specific documentation on how to connect to a variety of databases in the Configuration documentation ([ie. PostgresSQL](/../reference/configuration/Sources/PostgresqlSource/))
+    There is specific documentation on how to connect to a variety of databases in the Configuration documentation ([ie. PostgresSQL](/../reference/configuration/targets/PostgresqlSource/))
```
Comment (tgsoverly) [inline mkdocs/topics/sources.md:5]: Think this should be sources

**Review by tgsoverly — APPROVED** [review]

File: viewer/src/components/Dashboard.js [hunk 2]
```diff
@@ -16,13 +16,19 @@ const Dashboard = (props) => {
...
+            return 128
+        } else if (height === 'small') {
             return 256
         } else if (height === 'medium') {
             return 396
-        } else {
+        } else if (height === 'large') {
             return 512
+        } else if (height === 'xlarge') {
+            return 792
```
Comment (tgsoverly) [inline viewer/src/components/Dashboard.js:29]: Nit: 768 is more binary!

File: mkdocs/topics/sources.md (hunk 1 above)
Comment (jpjsonic) [reply mkdocs/topics/sources.md:5]: It should be sources. For some reason our parser is still labeling it as targets. I think maybe its coming from the aliasing? I didn't want to spend too much time digging into it, but it does bother me too

File: viewer/src/components/Dashboard.js (hunk 2 above)
Comment (jpjsonic) [reply viewer/src/components/Dashboard.js:29]: Oh good call will update.

File: mkdocs/topics/sources.md (hunk 1 above)
Comment (tgsoverly) [reply mkdocs/topics/sources.md:5]: Any chance it is the capitalization?

**Pull request merged**